### ISSUE TITLE
about the type of attribute-background that in interface CanvasConfig

### ISSUE THREAD

**Issue by notthistrain** [antvis/G#1331]
- [ ] I have searched the [issues](https://github.com/antvis/g/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Passing Parameters: we can pass CanvasPattern, and typescript do not report syntax error

### What does the proposed API look like?
finally this attribute is used for set fillStyle, so I think it can add a CanvasPattern type to background, like this, and it is worked I tried

**Comment by notthistrain** [comment]
https://github.com/antvis/G/blob/next/packages/g-lite/src/types.ts#L482

**Comment by xiaoiver** [comment]
> https://github.com/antvis/G/blob/next/packages/g-lite/src/types.ts#L482
> 
> > ```js
> > new Canvas({
> >   background: {
> >     image: 'http://example.png',
> >     repetition: 'repeat',
> >     transform: 'rotate(30deg)',
> >   }
> > });
> > ```
> 
> thanks for your answer, but i think it can not pass the typecheck

Yes, since I haven't implemented it yet. For now `background` only supports `string`.

**Comment by notthistrain** [comment]
thats ok,栓q
